### ISSUE TITLE
Water levels differ from the public tide predictors

### ISSUE THREAD

**Issue by landyrev** [neaps/tide-predictor#149]
Hey, @kevee! Thanks for a great lib.

I'm playing with tide predictions and noticed that water levels generated by neaps/tide-predictor are quite different from the ones I can find in other tide predictors.

As an example I'm comparing the Monterey, CA tidal station, 19.08.2022

Neaps results:
High level at `2022-08-19T21:08:11.000Z` is `0.711727...` meters
Low level at `2022-08-19T17:28:11.000Z` is `-0.0094...` meters

[US Harbors](https://www.usharbors.com/harbor/california/monterey-ca/tides/) results:
High level at `2022-08-19T00:13:00.000Z` is `1.55...` meters
Low level at `2022-08-19T17:31:00.000Z` is `0.85...` meters

So, while the time is pretty correct, the water levels looks offsetted compared to public predictions.

**Comment by kevee** [comment]
Thank you, @landyrev! I've been heads down in other projects for the past few months but I'll take a look at this PR next week.

Quick question - did you include an `offset` when you passed the harmonic constituents?

**Comment by Ub4tea** [comment]
Indeed, from a harmbase database you need to add the corresponding `data_sets.datum` as `offset`.

**Comment by noGAFAM** [comment]
@kevee, indeed, with `offset`, namely `data_sets.datum` from a `harmbase2` database it works. Any reason for not including this data by default?
